docs/core-manage-asdf-vm.md

```diff
@@ -277,29 +315,226 @@ asdf update --head
 brew upgrade asdf
 ```
 
+### -- Pacman --
+
+Refer to the [Arch Pacman upgrade documentation](https://wiki.archlinux.org/index.php/pacman#Upgrading_packages).
+
 <!-- select:end -->
 
 ## Remove
 
-Uninstalling `asdf` is as simple as:
+To unistall `asdf` follow these steps:
```

Comment:
Very minor issue, but `uninstall` is mispelled here.

Reply:
And of course I then misspelled misspelled :sweat_smile:

Reply:
I think we use different dictionaries :sweat_smile: Nonetheless, the change is more clear :pray: